crates/key-value-azure/src/store.rs

```diff
@@ -274,8 +329,9 @@ impl CompareAndSwap {
 
 #[async_trait]
 impl Cas for CompareAndSwap {
-    /// `current` will fetch the current value for the key and store the etag for the record. The
-    /// etag will be used to perform and optimistic concurrency update using the `if-match` header.
+    /// `current` will fetch the current value for the key and store the etag
```

Comment:
Not sure why this and other doc comments needed reformatting, or is it a new lint?

Reply:
I usually lint at 80 chars so must have updated this by accident. looks like we use 100.